utils/gitlab_api.py

```diff
@@ -102,6 +102,16 @@ def get_project_maintainers(self, repo_url):
     def get_app_sre_group_users(self):
         app_sre_group = self.gl.groups.get('app-sre')
         return [m for m in app_sre_group.members.list()]
+    
+
+    def get_group_members(self, group_name):
+        groups = self.gl.groups.list()
+        group_names = list(map(lambda x: x.name,groups))
+        if group_name not in group_names:
+            logging.info(group_name + " group not found")
```

Comment:
if the group is not found we just log and continue?

Reply:
I think that we should create the group when its not found, but I thought for this version there wasn't going to be anything involving creating and deleting groups or users, I just added this so that there wouldn't be an error if a group was not found.

Reply:
that is great thinking. these are the kinds of things that can be added as comments to explain your thought path. i'm with you on not failing, but i would suggest to use `logging.error` instead just so it stands out (it will not fail the integration).